liby2util-r/src/include/y2util/Ustring.h

```diff
@@ -128,12 +128,7 @@ class Ustring
  {
 
   private:
-
-    /**
-     * !!! It should actualy be const !!!
-     * But that way default copy and assingment can be used.
-     **/
-    std::string _name;
+    const std::string* _name;
```

Comment:
This will be a dangling pointer when the owning UstringHash goes away.
But grepping shows that it will in fact never go away because all Ustring users rely on a single static `SymbolEntry::_nameHash` UstringHash. Hmm.